Twig/Gettext/Extractor.php

```diff
@@ -35,10 +35,10 @@ class Extractor
 
     private $executable;
 
-    public function __construct(\Twig_Environment $environment)
+    public function __construct(\Twig_Environment $environment, $input = null)
     {
         $this->environment = $environment;
-        $this->reset();
+        $this->input = $input;
```

Comment:
Hang on, `$this->input` is never used, why do we have it here?

Reply:
I initially intended to use some input arguments (like debug) inside the Extractor.
Finally it was not the case, but that may useful later.
I've no problem with dropping it btw

Reply:
Let's drop it.